# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderTextureHost.h
# Commit: de8d00f1d270
# Full Hash: de8d00f1d270f43a27d9f7280105fe21c80461ad
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Support using D3D11 textures as external compositor surfaces with D3D11 compositor. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95800
# ==============================================================================

diff -r 9d637c05af7f -r de8d00f1d270 gfx/webrender_bindings/RenderTextureHost.h
--- a/gfx/webrender_bindings/RenderTextureHost.h	Fri Nov 06 00:01:36 2020 +0000
+++ b/gfx/webrender_bindings/RenderTextureHost.h	Fri Nov 06 00:01:54 2020 +0000
@@ -25,6 +25,7 @@
 
 class RenderCompositor;
 class RenderDXGITextureHost;
+class RenderDXGIYCbCrTextureHost;
 class RenderMacIOSurfaceTextureHost;
 class RenderBufferTextureHost;
 class RenderTextureHostSWGL;
@@ -69,6 +70,9 @@
   virtual bool SyncObjectNeeded() { return false; }
 
   virtual RenderDXGITextureHost* AsRenderDXGITextureHost() { return nullptr; }
+  virtual RenderDXGIYCbCrTextureHost* AsRenderDXGIYCbCrTextureHost() {
+    return nullptr;
+  }
 
   virtual RenderMacIOSurfaceTextureHost* AsRenderMacIOSurfaceTextureHost() {
     return nullptr;